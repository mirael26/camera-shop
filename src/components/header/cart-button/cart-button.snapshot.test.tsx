import { renderWithReduxAndRouter } from '../../../test/helpers/render-with-redux-and-router';
import CartButton from './cart-button';

test('CartButton component renders correctly', () => {
  const { asFragment } = renderWithReduxAndRouter(<CartButton/>);
  expect(asFragment()).toMatchSnapshot();
});
