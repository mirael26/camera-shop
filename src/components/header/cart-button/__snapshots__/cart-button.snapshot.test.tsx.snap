// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CartButton component renders correctly 1`] = `
<DocumentFragment>
  <a
    class="header__basket-link"
    href="/catalog/cart"
  >
    <svg
      aria-hidden="true"
      height="16"
      width="16"
    >
      <use
        xlink:href="#icon-basket"
      />
    </svg>
  </a>
</DocumentFragment>
`;
